aws-assume-role-policy/module_test.go

```diff
@@ -0,0 +1,62 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/chanzuckerberg/go-misc/tftest"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/require"
+)
+
+type oidc struct {
+	idp_arn    string
+	client_ids []string
+	provider   string
+}
+
+func TestAWSAssumeRolePolicy(t *testing.T) {
+	curAcct := tftest.AWSCurrentAccountID(t)
+	oidcCfg := []interface{}{}
+	oidcOptions := []oidc{
+		{
+			idp_arn:    "oidcARN1",
+			client_ids: []string{"clientID1", "clientID2", "clientID3"},
+			provider:   "testProvider",
+		},
+	}
+
+	for _, v := oidcOptions {
+		oidcCfg = append(oidcCfg, interface{
+			"idp_arn": v.idp_arn,
+			"client_ids": v.client_ids,
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>missing ',' in argument list (typecheck)